.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: python
 matrix:
   include:
-    - python: 2.7
+    - python: 3.6
+      dist: xenial
       services: xvfb
-    - python: 3.7
+    - python: 3.8
```

Comment:
Should we not continue to test 3.7 as well?

Reply:
We can continue to test all versions.  I'll run a test to see what that translates to in run time

Reply:
All three are running.  We can't run more than two parallel tasks, so total run time (six tests) for a pull request is ~1 hr 12 min.

Reply:
The python 3.8 tests take an extra five minutes or so.  I suspect this is because it is not auto-installed on travis yet.